examples/hello_ocean.jl

```diff
@@ -0,0 +1,35 @@
+# # Hello, ocean!
+#
+# > How inappropriate to call this planet Earth, when it is quite clearly _Ocean_.
+#
+#   --Arthur C. Clark
+
+using Oceananigans, Oceananigans.Grids, Plots
+
+grid = RegularCartesianGrid(size = (1, 64, 64),
+                            x = (0, 1), y = (-4, 4), z = (-4, 4),
```

Comment:
I don't like the choice of `-4<z<4`... Make it seems that something special happens at 4...
I'd vote for `z = (-4, 0)`

Reply:
just nice to have the origin at 0, 0, and the width of your signal be 1... hmmm

Reply:
oh I see... OK then
(otherwise also more characters will be needed for the initial condition ;))